README.rst

```diff
@@ -112,10 +112,12 @@ application and checks if it fails on an assertion might look like the one
 below::
 
     #! /bin/bash
-    timeout --foreground 10 <path/to/the/target/application> $1 2>&1 | grep -q "Assertion failed";
+    timeout --foreground 10 </path/to/the/target/application> $1 2>&1 | grep -q "Assertion failed";
```

Comment:
I'd not change this. The remark below should be enough. (Plus, we may even have a flag in the future that changes the behavior of `SubprocessTest` to make it _not_ change the current working directory when executing the test, which actually necessitates the use of absolute paths now.)